pyanaconda/core/string.py

```diff
@@ -0,0 +1,176 @@
+#
+# string.py - generic string utility functions
+#
+# Copyright (C) 2021  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import string  # pylint: disable=deprecated-module
```

Comment:
Could you remove `string` from `deprecated-modules` in our `pylintrc` file, please? It seems to be a Python 2 thing.